disperser/cmd/apiserver/Dockerfile

```diff
@@ -1,26 +0,0 @@
-FROM golang:1.21.1-alpine3.18 as builder
```

Comment:
Similar to previous comment, do we want to make the PR additive first and then deprecate this stuff later?

Reply:
How would you test this if we don't remove it? I feel let's just test it, and if for some reason the releases don't work or there's something super buggy with it, then we just git revert.

Reply:
We can still test it because there would be two separate pipelines. I'm fine with reverting if there's problems.